test-kitchen.gemspec

```diff
@@ -25,8 +25,8 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "mixlib-shellout", ">= 1.2", "< 3.0"
   gem.add_dependency "net-scp",         "~> 1.1"
-  gem.add_dependency "net-ssh", ">= 2.9", "< 5.0"
```

Comment:
We might want to constrain these to less than the next major. Who knows what comes in net-ssh 6.0 and net-ssh-gateway 3.0